### PR TITLE
[Device capabilities] [Location][1553134]

### PR DESCRIPTION
**Pull request by Rajeshwari-v**
Created location capability file and updated related files. Please review.

### CODE REVIEW

**Comment by Rajeshwari-v** [comment]
@Shubham chauhan, here, showMap conditions are little ambiguous. Please clarify.
image

File: msteams-platform/whats-new.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ The change log lists changes to the Microsoft Teams platform and this document s
...
 |02/09/2021|New: Added device capabilities overview. <br/> Update: Microphone capability information is added in the native device permissions and integrate media capabilities files.|[Overview](concepts/device-capabilities/device-capabilities-overview.md), [Request device permissions](concepts/device-capabilities/native-device-permissions.md), [Integrate media capabilities](concepts/device-capabilities/mobile-camera-image-permissions.md)|
 |11/30/2020|New: Identity platform integration with Teams Toolkit and Visual Studio Code for tabs|[Single sign-on authentication with Teams Toolkit and Visual Studio Code for tabs](toolkit/visual-studio-code-tab-sso.md)|
```
Comment (shuchauh) [inline msteams-platform/whats-new.md:21]: This is great. Didn't know we've been updating the "What's new in Teams" change log as well.

File: msteams-platform/concepts/device-capabilities/location-capability.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
+---
+title: Integrate location capability
+description: How to use Teams JavaScript client SDK to leverage location capability
+keywords: camera media qr code qrcode bar code barcode scanner scan capabilities native device permissions location
```
Comment (shuchauh) [inline msteams-platform/concepts/device-capabilities/location-capability.md:4]:
```suggestion
keywords: location map capabilities native device permissions
```

File: msteams-platform/concepts/device-capabilities/location-capability.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+ms.author: lajanuar
+---
+
+# Integrate location capability 
+
+This document guides you on how to integrate the **location** capability of native device with your Teams app.  
+
+You can use [Microsoft Teams JavaScript client SDK](/javascript/api/overview/msteams-client?view=msteams-client-js-latest&preserve-view=true), which provides the tools necessary for your app to access the user’s [native device capabilities](native-device-permissions.md). Use the location APIs, such as [getLocation](/javascript/api/@microsoft/teams-js/location?view=msteams-client-js-latest#getLocation_LocationProps___error__SdkError__location__Location_____void_) and [showLocation](/javascript/api/@microsoft/teams-js/location?view=msteams-client-js-latest#showLocation) to integrate the capability within your app.
```
Comment (shuchauh) [inline msteams-platform/concepts/device-capabilities/location-capability.md:12]:
```suggestion
You can use [Microsoft Teams JavaScript client SDK](/javascript/api/overview/msteams-client?view=msteams-client-js-latest&preserve-view=true), which provides the tools necessary for your app to access the user’s [native device capabilities](native-device-permissions.md). Use the location APIs, such as [getLocation](/javascript/api/@microsoft/teams-js/location?view=msteams-client-js-latest#getLocation_LocationProps___error__SdkError__location__Location_____void_) and [showLocation](/javascript/api/@microsoft/teams-js/location?view=msteams-client-js-latest#showLocation_Location___error__SdkError__status__boolean_____void_) to integrate the capability within your app. 
```

File: msteams-platform/concepts/device-capabilities/location-capability.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+You can use [Microsoft Teams JavaScript client SDK](/javascript/api/overview/msteams-client?view=msteams-client-js-latest&preserve-view=true), which provides the tools necessary for your app to access the user’s [native device capabilities](native-device-permissions.md). Use the location APIs, such as [getLocation](/javascript/api/@microsoft/teams-js/location?view=msteams-client-js-latest#getLocation_LocationProps___error__SdkError__location__Location_____void_) and [showLocation](/javascript/api/@microsoft/teams-js/location?view=msteams-client-js-latest#showLocation) to integrate the capability within your app. 
+
+## Advantages of integrating location capability
+
+* The integration allows web app developers on Teams platform to leverage location functionality with Microsoft Teams JavaScript client SDK. 
+
+Following examples show how the integration of **location** capability is used in different scenarios:
+ * In a factory, the supervisor can track the attendance of workers by asking them to take a selfie in the vicinity of the factory and share it through the specified app.
```
Comment (shuchauh) [inline msteams-platform/concepts/device-capabilities/location-capability.md:19]:
```suggestion
 * In a factory, the supervisor can track the attendance of workers by asking them to take a selfie in the vicinity of the factory and share it through the specified app.  The location data also gets captured and sent along with the image.
```

File: msteams-platform/concepts/device-capabilities/location-capability.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+## Advantages of integrating location capability
+
+* The integration allows web app developers on Teams platform to leverage location functionality with Microsoft Teams JavaScript client SDK. 
+
+Following examples show how the integration of **location** capability is used in different scenarios:
+ * In a factory, the supervisor can track the attendance of workers by asking them to take a selfie in the vicinity of the factory and share it through the specified app. 
+* The **location** capability enables the maintenance staff of a service provider to share authentic health data of cellular towers with the management. The management can track any mismatch of captured location information with actual data through a custom-built app.
```
Comment (shuchauh) [inline msteams-platform/concepts/device-capabilities/location-capability.md:20]:
```suggestion
* The **location** capability enables the maintenance staff of a service provider to share authentic health data of cellular towers with the management. The management can compare any mismatch between captured location information and the data submitted by maintenance staff.
```

File: msteams-platform/concepts/device-capabilities/location-capability.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+
+You must use the following set of APIs to enable your device's location capability:
+
+| API      | Description   |
+| --- | --- |
+|[getLocation](/javascript/api/@microsoft/teams-js/location?view=msteams-client-js-latest#getLocation_LocationProps___error__SdkError__location__Location_____void_) | Gives user’s current device location or opens native location picker and returns the location chosen by the user. |
+|[showLocation](/javascript/api/@microsoft/teams-js/location?view=msteams-client-js-latest#showLocation) | Shows location on map |
+
+> [!NOTE]
+> The user location is displayed based on the value of location properties, `allowChooseLocation` and `showMap`. <br/> If the value of `allowChooseLocation` is *true*, then the users can choose any location of their choice.<br/>  If the value is *false*, then the users cannot change their current location.<br/> If the value of `showMap` is *true*, the map displays the current location of the user. <br/>
```
Comment (shuchauh) [inline msteams-platform/concepts/device-capabilities/location-capability.md:51]:
```suggestion
> In getLocation() API comes along with following [input configurations](https://docs.microsoft.com/en-us/javascript/api/@microsoft/teams-js/locationprops?view=msteams-client-js-latest), `allowChooseLocation` and `showMap`. <br/> If the value of `allowChooseLocation` is *true*, then the users can choose any location of their choice.<br/>  If the value is *false*, then the users cannot change their current location.<br/> If the value of `showMap` is *false*, the current location is fetched without displaying the map. `showMap` is ignored if `allowChooseLocation` is set to *true*.<br/>
```

File: msteams-platform/concepts/device-capabilities/location-capability.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+> If the value of `allowChooseLocation` is *false*, the default value of `showMap` is also *false*.
```
Comment (shuchauh) [inline msteams-platform/concepts/device-capabilities/location-capability.md:52]:
```suggestion
```

**Review by shuchauh — CHANGES_REQUESTED** [review]

**Review by shuchauh — APPROVED** [review]

**Review by shuchauh — APPROVED** [review]
Reviewed and approved

**Review by KirtiPereira — APPROVED** [review]
Approved

**Review by surbhigupta12 — APPROVED** [review]
Approving the PR

**Pull request merged**